docs/pages/components/form.md

```diff
@@ -92,25 +92,25 @@ Do not use the input field if:
     </div>
     <br />
     <br />
-    <div class="fd-form-item fd-form-item--horizontal">
+    <div class="fd-form-item--horizontal">
         <label class="fd-form-label" for="input-1a">Default Input:</label>
         <input class="fd-input" type="text" id="input-1a" placeholder="Field placeholder text">
     </div>
     <br />
     <br />
-    <div class="fd-form-item fd-form-item--horizontal">
+    <div class="fd-form-item--horizontal">
```

Comment:
Is it good usage of BEM things? I see `fd-form-item` gives `fd-reset()` Normally it should be `fd-form-item fd-form-item--horizontal`